sriov/sriov.go

```diff
@@ -82,6 +82,11 @@ func moveIfToNetns(ifname string, netns ns.NetNS) error {
 		return fmt.Errorf("failed to lookup vf device %v: %q", ifname, err)
 	}
 
+	netlink.LinkSetDown(vfDev)
+	index := vfDev.Attrs().Index
+	vfName := fmt.Sprintf("dev%d", index)
+	renameLink(ifname, vfName)
```

Comment:
Can we capture the err of renameLink if any? Just like how renameLink is used in other places.

Reply:
Should we return the error from LinkSetDown()? Or continue with the renaming?
If we decide to ignore the error(for link already is down, not sure how this function behaves) then we should capture this error with blank identifier(_).

I'm seeing this same change on #45 https://github.com/intel/sriov-cni/pull/45/files#diff-8becae7a8222be03e1bcd86d9adaa117R85

Please, introduce changes single logical change at a time per PR by not repeating them on multiple PR. Thanks